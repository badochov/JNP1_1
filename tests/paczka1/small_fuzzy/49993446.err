Error in line 2: S43 A653 35,9
Error in line 1: lNaRSbxbR S121 35,7
Error in line 13: woTmw S121 77,1
