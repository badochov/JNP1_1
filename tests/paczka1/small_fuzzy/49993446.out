woTmw S 2,0
